src/com/amazon/ion/benchmark/Main.java

```diff
@@ -66,6 +68,7 @@
             + "DOM loader is included in each timed benchmark invocation. Therefore, it is important to provide "
             + "data that closely matches the size of data read by a single reader/loader instance in the real "
             + "world to ensure the initialization cost is properly amortized.\n"
+        + " generate\n"///
```

Comment:
The description will eventually need to be filled in. We can work together to come up with the best description for the `generate` command.